logfire/_internal/main.py

```diff
@@ -1223,6 +1263,30 @@ def instrument_redis(self, **kwargs: Unpack[RedisInstrumentKwargs]) -> None:
         self._warn_if_not_initialized_for_instrumentation()
         return instrument_redis(**kwargs)
 
+    def instrument_mysql(
+        self,
+        conn: PooledMySQLConnection | MySQLConnectionAbstract | None = None,
+        **kwargs: Unpack[MySQLInstrumentKwargs],
+    ) -> PooledMySQLConnection | MySQLConnectionAbstract | None:
```

Comment:
```suggestion
        conn: MySQLConnection = None,
        **kwargs: Unpack[MySQLInstrumentKwargs],
    ) -> MySQLConnection:
```

where `MySQLConnection = TypeVar('MySQLConnection', PooledMySQLConnection, MySQLConnectionAbstract, None)`